scripts/reloader.bash

```diff
@@ -11,12 +11,15 @@ function _set-prefix-based-on-path()
 
 if [ "$1" != "skip" ] && [ -d "./enabled" ]; then
   _bash_it_config_type=""
-  if [[ "${1}" =~ ^(alias|completion|plugin)$ ]]; then
-    _bash_it_config_type=$1
-    _log_debug "Loading enabled $1 components..."
-  else
-    _log_debug "Loading all enabled components..."
-  fi
+
+  case $1 in
+    alias|completion|plugin)
+      _bash_it_config_type=$1
+      _log_debug "Loading enabled $1 components..." ;;
+    *|'')
```

Comment:
why the `''` here? isnt the glob enough?

Reply:
The `''` is for matching a non-value. Technically, you could probably just put `*|)` but I like to be explicit so the reader knows my goal.

Reply:
shellcheck actually complains about this, see the following warnings- [one](https://www.shellcheck.net/wiki/SC2221) + [two](https://www.shellcheck.net/wiki/SC2222)

I prefer that you remove it

Reply:
Okay, I'll remove it. It's harmless, BTW.